BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/Onboarding/DigitalInvoiceOnboardingViewController.swift

```diff
@@ -73,8 +73,7 @@ final class DigitalInvoiceOnboardingViewController: UIViewController {
     // swiftlint:disable function_body_length
     private func configureUI() {
         let configuration = GiniBankConfiguration.shared
-        title = NSLocalizedStringPreferredFormat("ginibank.digitalinvoice.screentitle",
```

Comment:
Just to make sure, we didn't removed this by mistake. Why we don't need to set the title also here?

Reply:
@ValentinaIancu-Gini we don't show the title on the Onboarding screen
https://www.figma.com/file/m4OtCvxWxPMeB0Z5k4Aeow/Gini-Photo-Payment-%2F-iOS?type=design&node-id=1094%3A21729&t=ea1R2PUKe1MAPqsl-1